SalesforceSDK/Salesforce.SDK.Phone/Source/Pages/AccountPage.xaml

```diff
@@ -134,7 +126,7 @@
                    Text="{Binding Converter={StaticResource Localize}}"
                    Margin="5"
                    Style="{StaticResource HeaderTextBlockStyle}" />
-        <ItemsControl x:Name="ApplicationLogo">
+        <ItemsControl x:Name="ApplicationLogo" AutomationProperties.Name="Salesforce1">
```

Comment:
The property should be more generic, matching more closely the name itself.  It shouldn't be named as a specific application.